### DISCUSSION TITLE
Several error messages

### DISCUSSION THREAD

**Discussion by AlexandreOnnee** [tmijail/Odoo-Power-BI-Connector#21] [Q&A]
Hi Tmijail,

Many thanks for you works and this connector.
I use Odoo online and here are the three error messages I get.
Do you have any idea how to solve them?

Thank you very much in advance for your help!

![Errors messages](https://github.com/tmijail/Odoo-Power-BI-Connector/assets/135993274/4d29e7c9-94c0-4bc4-bf37-d69178eef70c)

**Comment by tmijail** [comment]
First two errors are fixed in the new releases [v1.0.0](https://github.com/tmijail/Odoo-Power-BI-Connector/releases/tag/v1.0.0) and [v0.2.0](https://github.com/tmijail/Odoo-Power-BI-Connector/releases/tag/v0.2.0). 

The others appear to be permissions errors

**Comment by AlexandreOnnee** [comment]
It works! Fantastic!
Thank you so much for solving the problem.
